lib/view/channels-panel.js

```diff
@@ -13,7 +13,7 @@ class ChannelsPanel {
     this.selectedChannelItem = null
 
     this.view = gui.Container.create()
-    this.view.setStyle({width: 180})
+    this.view.setStyle({width: 225})
```

Comment:
This is slightly larger than the official app.
<img width="306" alt="screen shot 2018-04-10 at 19 36 56" src="https://user-images.githubusercontent.com/639601/38552614-a376ce76-3cf7-11e8-8ed7-34bf11783f2d.png">